x-pack/plugins/cases/server/saved_object_types/comments.ts

```diff
@@ -91,15 +91,28 @@ export const createCaseCommentSavedObjectType = ({
           },
         },
       },
-      externalReferenceMetadata: {
+      persistableStateAttachmentState: {
         dynamic: false,
         properties: {},
       },
-      persistableStateAttachmentState: {
+      */
+      externalReferenceMetadata: {
         dynamic: false,
-        properties: {},
+        properties: {
+          files: {
+            properties: {
+              name: {
+                type: 'text',
```

Comment:
We don't actually need this at the moment but I don't know if this would cause a breaking change so I'm mapping `name` as text and `text.fields.keyword` as keyword. The keyword is needed for the `value_count` aggregation.

Reply:
Is it possible to use the file service to get the count of files attached to the specific case?